### DISCUSSION TITLE
Plan to translate documents into Simplified Chinese?

### DISCUSSION THREAD

**Discussion by GuoJikun** [slint-ui/slint#3206] [General] [answered]
Plan to translate documents into Simplified Chinese?

**Comment by tronical** [comment]
Hi! At the moment I don't think we're planning on translating the documentation. That's an effort we're not equipped for at this point, unfortunately.

Or did you mean something else with "documents"?

**Reply by GuoJikun** [reply]
![image](https://github.com/slint-ui/slint/assets/21582741/4334493e-db60-4d39-a977-511ca11ec984)

**Reply by tronical** [reply, marked as answer]
Thanks for the clarification. I think we'd like to translate the website eventually, but I don't think that we're going to translate the API documentation at this point. :-(
